### DISCUSSION TITLE
Plots Created with Public Key

### DISCUSSION THREAD

**Discussion by g1thub11** [Chia-Network/chia-blockchain#7155] [Support]
I created some plots with the public key, instead of the farmer public key. The plots are valid, but seemingly not farmable. Is there a way to farm plots created with the public key?

**Comment by loppefaaret** [comment]
not if you pointed it towards a plot NFT

**Reply by g1thub11** [reply]
I have not knowingly pointed anything towards a plot NFT.

Does that mean it's possible to farm the plots created with the public key?

**Reply by loppefaaret** [reply]
oh, it is OG plots, with switched public & farmers keys ? im not sure it is going to work anyways though...

**Reply by g1thub11** [reply]
"oh, it is OG plots, with switched public & farmers keys ?"

Yes.

**Reply by loppefaaret** [reply]
i doubt yu are going to successfully farm those plots

**Comment by BrandtH22** [comment]
Since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
